src/plugins/data/public/antlr/opensearch_sql/code_completion.ts

```diff
@@ -44,7 +44,33 @@ export const getSuggestions = async ({
     if (suggestions.suggestColumns?.tables?.length) {
       // NOTE:  currently the suggestions return the table present in the query, but since the
       //        parameters already provide that, it may not be needed anymore
-      finalSuggestions.push(...fetchFieldSuggestions(indexPattern));
+      finalSuggestions.push(...fetchFieldSuggestions(indexPattern, (f: any) => `${f} = `));
+    }
+
+    if (suggestions.suggestValuesForColumn) {
+      // get dataset for connecting to the cluster currently engaged
+      const dataset = services.data.query.queryString.getQuery().dataset;
+
+      const res = await fetchColumnValues(
+        [indexPattern.title],
+        suggestions.suggestValuesForColumn,
+        services,
+        dataset
+      );
+
+      let i = 0;
+      finalSuggestions.push(
+        ...res.body.fields[0].values.map((val: any) => {
+          i++;
+          return {
+            text: val.toString(),
+            insertText: typeof val === 'string' ? `"${val}" ` : `${val} `,
```

Comment:
have we tested number value?

Reply:
yeah so i have the logic like this since if the type isn't of type string, such as number, it should be unwrapped of quotes. i haven't seen any other data types that can be aggregated other than string or number, do you know if there are any others?